percona/server/install.sls

```diff
@@ -8,10 +8,10 @@ include:
   - percona.custom_version
 
 {% set mysql = salt['grains.filter_by'](rawmap, grain='os', merge=salt['pillar.get']('percona:lookup')) %}
-{% set mysql_root_user = salt['pillar.get']('percona:server:root_user', 'root') %}
-{% set mysql_root_password = salt['pillar.get']('percona:server:root_password', salt['random.get_str'](32)) %}
-{% set mysql_host = salt['pillar.get']('percona:server:host', 'localhost') %}
-{% set defaults_extra_file = salt['pillar.get']('percona:defaults_extra_file', mysql.defaults_extra_file) %}
+{% set mysql_root_user = salt['pillar.get']('mysql:server:root_user', 'root') %}
+{% set mysql_root_password = salt['pillar.get']('mysql:server:root_password', salt['random.get_str'](32)) %}
```

Comment:
The template fails to render here, I think it doesn't like mysql:server:root_user in the value, though that's odd because it worked ok with the previous value. 

I am using mysql_root_password instead, since that's set earlier on this file.